src/components/Book.jsx

```diff
@@ -2,7 +2,7 @@ import { CircularProgressbar } from "react-circular-progressbar";
 import "react-circular-progressbar/dist/styles.css";
 import PropTypes from "prop-types";
 import { useDispatch } from "react-redux";
-import { removeBooks } from "../redux/books/BookSlice";
+import deleteBook from "../redux/books/BookSlice";
```

Comment:
- [ ] Also, kindly fix MIT license link in your readme. Because, If I click the link it leads me to [404 page](https://github.com/Simpleshaikh1/Book-Store/blob/use-redux-in-react-component/MIT.md).